hack/release.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage:
+#   hack/release.sh $VERSION
+#
+# Example:
+#   hack/release.sh v0.2
+
+VERSION=$1
+
+KO_ROOT="$(cd "$(dirname "$0")" && pwd)/.."
+govvv build -o $KO_ROOT/build/ko $KO_ROOT/cmd/ko -version $VERSION
```

Comment:
Perhaps add a `go get github.com/ahmetb/govvv` before this?

Reply:
Good point. Added.